packages/terra-application-utility/src/utility/_UtilityMenu.module.scss

```
@@ -19,19 +19,19 @@
   }
 
   .header-utility-menu-header {
-    color: var(--terra-application-header-utility-menu-header-color);
-    font-size: var(--terra-application-header-utility-menu-header-font-size, 1rem);
-    font-weight: var(--terra-application-header-utility-menu-header-font-weight, normal);
-    line-height: var(--terra-application-header-utility-menu-header-line-height, 1.2);
-    margin-top: var(--terra-application-header-utility-header-margin-top, 0.7143rem);
+    color: var(--terra-application-utility-header-menu-header-color);
+    font-size: var(--terra-application-utility-header-menu-header-font-size, 1rem);
+    font-weight: var(--terra-application-utility-header-menu-header-font-weight, normal);
+    line-height: var(--terra-application-utility-header-menu-header-line-height, 1.2);
+    margin-top: var(--terra-application-utility-header-header-margin-top, 0.7143rem);
   }
 
   .menu-utility-menu-header {
-    color: var(--terra-application-header-utility-menu-menu-color);
-    font-size: var(--terra-application-menu-utility-menu-header-font-size, 1rem);
-    font-weight: var(--terra-application-menu-utility-menu-header-font-weight, normal);
-    line-height: var(--terra-application-header-utility-menu-header-line-height, 1.2);
-    margin-top: var(--terra-application-menu-utility-header-margin-top, 0.7143rem);
+    color: var(--terra-application-utility-header-menu-menu-color);
+    font-size: var(--terra-application-utility-menu-menu-header-font-size, 1rem);
+    font-weight: var(--terra-application-utility-menu-menu-header-font-weight, normal);
+    line-height: var(--terra-application-utility-header-menu-header-line-height, 1.2);
+    margin-top: var(--terra-application-utility-menu-header-margin-top, 0.7143rem);
```

These seem to be inconsistent, should these have a `utility-menu` or a `utility-header` substring?

---

These should all be `utility-menu` I think. I assume it never got touched because it was set like that to begin with and just was never fixed.

---

let's update these to `utility-menu`
```suggestion
    margin-top: var(--terra-application-utility-menu-menu-line-height, 0.7143rem);
```

---

Fixed in https://github.com/cerner/terra-framework/pull/429/commits/e60c3d30ffddf82552113946c4f687d2a86f6fb1